configuration/pih/scripts/global/zl.js

```diff
@@ -87,3 +87,33 @@ function setUpExpandableContacts(badPhoneNumberMsg) {
   }
 
 }
+
+function showDateIfCheckboxSelected(){
```

Comment:
This looks like what the `toggle` attribute of the [obs tag](https://wiki.openmrs.org/display/docs/HTML+Form+Entry+Module+Reference#HTMLFormEntryModuleReference-%3Cobs%3E) is supposed to do.

Reply:
@lnball @brandones 
Of course, but it didn't work for the checkbox  or maybe the answerConceptId doesn't match. that's why I created this function.